css-images-4/Overview.bs

```diff
@@ -1897,15 +1894,49 @@ Color Stop “Fixup”</h4>
 		</pre>
 	</div>
 
-
-
+<!--
+ ██████  ████████ ████████  ████ ████████  ███████  ██████    ███ ███
+██    ██    ██    ██     ██  ██  ██     ██ █       ██    ██  ██     ██
+██          ██    ██     ██  ██  ██     ██ █       ██       ██       ██
+ ██████     ██    ████████   ██  ████████  █████    ██████  ██       ██
+      ██    ██    ██   ██    ██  ██        █             ██ ██       ██
+██    ██    ██    ██    ██   ██  ██        █       ██    ██  ██     ██
+ ██████     ██    ██     ██ ████ ██        ███████  ██████    ███ ███
+-->
 
 1D Image Values: the ''stripes()'' notation {#stripes}
 ======================================================
 
-	Issue: <a href="https://github.com/w3c/csswg-drafts/issues/2532">Per WG resolution</a>,
-	define the ''stripes()'' function
-	which creates a 1D image for use in borders/outlines.
+	The ''stripes()'' function allows to define a number of stripes of
+	different colors with different widths.
+
+	The syntax for ''1d-image'' and ''stripes()'' is defined as:
+
+	<pre class=prod>
+		<dfn>&lt;1d-image></dfn> = <<stripes()>>
+		<dfn>stripes()</dfn> = stripes( [ <<color>> && [ <<length-percentage>> | <<flex>>]? ]# )
+	</pre>
+
+	The function represents a 1-dimensional image generated by a list of colors
+	and an optional thickness for each of them.
+
+	A <<percentage>> or <<flex>> value refers to the total length of the image
+	in the used context.
+	The <<percentage>> must be between ''0%'' and ''100%'' inclusive;
+	any other value is invalid.
+
+	If the thickness of a stripe is omitted, it is interpreted as ''1fr''.
+
+	If not defined differently by the context the function is used in, the
+	stripes are painted from top to bottom and tiled horizontally. If the size
+	of the painting area exceeds the total thickness of the stripes, the
+	remaining area is painted transparently. I.e. it behaves as if there was an
+	additional last ''transparent'' stripe. If the size of the painting area
+	is smaller than the total thickness of the stripes, the stripes are clipped
+	starting from the last defined color.
+
+	The computed value of this function is the [=specified value=] with all
+	<<color>>s computed '1fr' values removed.
```

Comment:
No, we don't want to remove the fr values. :) We just want lengths and colors computed. So the computed value of this function is a list, each item pairing a computed color and either a `<<flex>>` value or a computed `<<length-percentage>>`.

Reply:
By saying that `1fr` is removed I wanted to accomodate the serialization rules which say to omit component values for a shorter serialization when they don't change the meaning. And I defined that an omitted thickness is equivalent to `1fr`.
Though I guess it's not necessary to explicitly mention that as that's already part of the serialization rules.

I'll adjust the wording according to your suggestion.

Reply:
Computed value is not serialization! The two concepts are completely unrelated, tho they were often conflated in older spec text. In this case, every stripe has a color and a thickness; the fact that a `1fr` thickness is omitted in the serialization falls out automatically from the shortest serialization principle.

Reply:
I guess that's covered by Tab's changes already.

Sebastian